app/views/mailers/administrator_notifications/channel_notifications_mailer/erroneous_import_records.liquid

```diff
@@ -0,0 +1,10 @@
+<p>Hello,</p>
+
+<p>Your contacts import upload failed, please check the attached file for the failed records. </p>
+<p>Please check the contacts tab.</p>
+
```

Comment:
lets include the count of how many records were successfully imported and how many failed . ref: https://linear.app/chatwoot/issue/CW-1362#comment-97daed6c

Also lets go with the something like following copy 

```
Your contact import has been completed. Please check the contacts tab to view the imported contacts
Number of records imported: count
{% if any records filed %}
Number of record failed: count
attachments
{% end % }

```